eventually/tests/unittests/curriculum/testcurriculummodel.py

```diff
@@ -22,36 +22,34 @@ def setUp(self):
             mock_time.return_value = TEST_DATE
 
             self.custom_user = CustomUser.objects.create(id=1,
```

Comment:
Do not use `self` to create instances if it is not required

Reply:
done!